lib/device-log/ios-simulator-log.js

```diff
@@ -102,8 +103,8 @@ class IOSSimulatorLog extends IOSLog {
     return this.proc && this.proc.isRunning;
   }
 
-  onOutput (prefix = '') {
-    const logs = this.logRow.split('\n');
+  onOutput (logRow, prefix = '') {
+    const logs = _.cloneDeep(logRow).split('\n');
```

Comment:
I'd rather clone the result of split, because internal split does not create new string instances, but rather pointers to the original big string.

Reply:
Done